tutorials/04_density_estimators.ipynb

```diff
@@ -34,7 +34,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "inference = SNPE(simulator, prior, density_estimator='maf')"
+    "inference = SNPE(prior, density_estimator='maf')"
```

Comment:
This applies to all tutorials: I would suggest being explicit for all keywords whenever creating the inference object, i.e., even write `prior=prior` which will avoid errors